lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -181,6 +181,8 @@ def get_task_info(subtask)
         workflow_at_version.tasks.find do |key, task|
          key == subtask["task"]
         end.try(:last) || {}
+      rescue
```

Comment:
When the subtask isn't a hash, from the specs `{ "task"=>"T3", "value"=>0 }` so:
```
> subtask = 0
 => 0 
> subtask["task"]
TypeError: no implicit conversion of String into Integer
``